lib/ClimaLandSimulations/src/utilities/fluxnet_simulation.jl

```diff
@@ -0,0 +1,15 @@
+"""This file contains the site-generic time variables for running CliMALSM on 
```

Comment:
since some of these are very specific to fluxnet simulations, should these be moved into src/fluxnet?

Reply:
I think this file actually just defines timestepper etc. and should be general, not fluxnet specific - so I renamed it and left it in utilities